deploy/apex/overlays/dev/kustomization.yaml

```diff
@@ -109,6 +109,13 @@ patchesJson6902:
     - op: add
       path: /spec/template/spec/volumes
       value: [{"name":"trust","secret":{"secretName":"apex-ca-key-pair","optional":false,"items":[{"key":"ca.crt","path":"tls-ca-bundle.pem"}]}}]
+- target:
```

Comment:
Could you add this to the configmap at `deploy/apex/base/apiserver/kustomization.yaml` instead with value of `false`... The deployment.yaml would set that environment variable to the valueFrom configmap... and in this kustomization.yaml you simply override the configmap value like we do for hostnames already...

Yeah it's a little obscure, but there is method in the madness...
When I'm back from PTO, we'll be adding a new `configMapGenerator` to these `overlays` which has a `strategy: merge`... that will merge values in our configMap with the one in base... and we won't need to use patches to set these values 😄

Reply:
Sounds good, I'll move it

Reply:
I changed it to use the configmap merge strategy